indico/modules/auth/forms.py

```diff
@@ -52,6 +54,13 @@ def _check_not_email_address(form, field):
         raise ValidationError(_('Your username cannot be an email address.'))
 
 
+def _check_signin(form, field):
+    signal_response = values_from_signal(signals.users.handle_signin.send(field.data), as_list=True)
+    if signal_response and not signal_response[0][0]:
```

Comment:
This will not work properly if more than one plugin handles this signal.

Reply:
Ok, I'll test it with multiple calls and fix the code.